packit/utils/koji_helper.py

```diff
@@ -14,18 +14,54 @@
 logger = logging.getLogger(__name__)
 
 
-class KojiHelper:
-    """
-    Class for querying Koji.
+class SessionWrapper:
+    def __init__(self) -> None:
+        self.session = self._open_session()
+
+    def __getattr__(self, name: str) -> Callable:
+        if name in self.__dict__:
+            return self.__dict__[name]
```

Comment:
If I get it correctly, this is implementing (probably as a side effect) a sort of cache. I am wondering if this could be somehow dangerous?

Reply:
What do you mean? It just returns local attributes (`session` and `SessionWrapper` methods).

Reply:
oh right 🙈